Felienne/hedy
# [FIX] Re-direct class related pages through for-teachers for correct active menu tab

Provide a general summary of your changes in the Title of the PR

**Description**
Currently the `class-overview` and `customize-class` pages are "stand-alone urls, starting directly from root. With the new structure of retrieving the active page the first part of the url is retrieved and selected as active in the menu bar. Due to these pages being a subcategory of `for-teachers`, no active tab is shown. To solve this we make all class related pages a sub url of `for-teachers`. So, instead of `/class/<class_id>` we now navigate to `/for-teachers/class/<class_id>`.

**Fix for**
This PR fixed #1373.

**How to test**
Navigate through all class related pages and notice that the "for-teachers" tab is correctly shown as active in the menubar. Also verify that all pages still work correctly.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
